cogs/Verification.py

```diff
@@ -44,6 +50,13 @@ def verify_is_enabled(ctx):
 
     return result or (str(ctx.author) == KoalaBot.TEST_USER and KoalaBot.is_dpytest)
 
+
+def check_if_role_exists(guild, university):
```

Comment:
Please add a docstring above this, even though it's self explanitory it's still worth having one